### PR TITLE
Rockets structure

### LINKED ISSUE #42: [1pt] Create basic structure for Rockets - Setup
- Create a route and a view component. Use `<NavLink />` for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- Create a directory for all Redux state slice files.

### PR DESCRIPTION
**Pull request by GoddessNgozi**
## In this PR I implemented the following:

- Create a route and a view component. Use <NavLink /> for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- Create a directory for all Redux state slice files.

### CODE REVIEW

File: src/components/Rockets/Rocket.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const Rocket = ({description, title, image, id}) => {
+  
+    return (
+      <div>
+        <div>
+          <img src={image} alt="" />
```
Comment (adel-gu) [inline src/components/Rockets/Rocket.jsx]: - [ ] For browser accessibility please add an alternative text to the rocket image, you can get it from `{title}`.

**Review by adel-gu — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻

Hi @GoddessNgozi 👋 great job so far, well done with creating a sub-component `rocket.jsx`😎,
However, there are some suggestions that would help your project make it to the next level 🚀.



## Highlights 
- Redux rocket slice setup ✔
- Setup the rockets list component ✔

## Required changes ♻
- [ ] Please fix linter errors.

**Review by adel-gu — APPROVED** [review]
# Status: APPROVED 🟢
I appreciate the great work you've accomplished 😎, well done 👏👏👏 There is nothing else to say other than... it's time to merge it :shipit:

image

## Highlights 
- No linter errors ✔

_If you have any other questions please feel free to leave your comments in the PR thread_

**Comment by GoddessNgozi** [comment]
@adel-gu Thank you

**Pull request merged**